q2_cutadapt/tests/test_demux.py

```diff
@@ -294,6 +294,40 @@ def test_di_typical(self):
                          b'@id6\nTTTTTGCATGCA\n+\nzzzzzzzzzzzz\n']
         self.assert_untrimmed_results(exp_untrimmed, obs_untrimmed_art)
 
+    def test_multiple_orientations(self):
```

Comment:
Can you add another test that matches the schema defined here: https://cutadapt.readthedocs.io/en/stable/guide.html?demultiplexing-paired-end-reads-in-mixed-orientation

To clarify, the new test would not be dual barcoded, rather the barcode is either on the forward, or, on the reverse.

Reply:
I'm not sure I understand what you're asking for, that page talks about a lot of things, and the part about demuxing mixed orientations looks like what the test already does.

Reply:
There was a typo in the URL, check again.

Reply:
Please pay close attention to the "read layout" example in the cutadapt docs linked above - from what I can tell, this test does not look like it replicates the read layout example that the cutadapt authors have specified. It is important that we make sure we are able to handle the case where the reads are not dual barcoded. As a reminder, dual-barcoding is when a sample is identified by the combination of barcode in the forward and reverse read, as opposed to having a barcode only in the forward read or only in the reverse read.

Reply:
Ah ok, yeah this makes sense now. Thank you. So we want to add tests with adapters only at the beginning or only at the end of the read? Do we also want tests with adapters in the middle of the read?